### PR TITLE
extract img crop for list of posts from new-html-css branch

### PR DESCRIPTION
**Pull request by AucT**
## Summary by CodeRabbit

- **New Features**
	- Enhanced image handling in blog post layouts for improved performance and user experience.
	- Introduced a new responsive image rendering template that supports multiple formats.

- **Bug Fixes**
	- Streamlined image retrieval process to ensure consistent display of images across blog posts.

### CODE REVIEW

File: themes/beaver/layouts/partials/blog/img-cropped.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+<picture>
+    <source srcset="{{ $webp.Permalink }}" type="image/webp">
+    <source srcset="{{ $cropped.Permalink }}" type="{{ $cropped.MediaType }}">
+    <img src="{{ $cropped.Permalink }}" alt="{{ .Page.Title }}" width="{{ .width }}" height="{{ .height }}">
+</picture>
+
+{{ else }}
+<img src="/img/no-image.svg" alt="Placeholder Image" width="{{ .width }}" height="{{ .height }}">
```
Comment (pftg) [inline themes/beaver/layouts/partials/blog/img-cropped.html]: do not use hard coded static path, use processed path or better inline svg

Comment (AucT) [reply themes/beaver/layouts/partials/blog/img-cropped.html]: we don't need to process svg
we don't need to inline svg if that svg will be used 100 times

Comment (AucT) [reply themes/beaver/layouts/partials/blog/img-cropped.html]: nDEU72hACa

Comment (pftg) [reply themes/beaver/layouts/partials/blog/img-cropped.html]: @AucT, we process (not convert images, but file renaming) all files, including SVG, for caching and optimization purposes. Still need to have processing: fingerprint

Comment (AucT) [reply themes/beaver/layouts/partials/blog/img-cropped.html]: ++

**Pull request merged**